docs/provision_hypershift_clusters_by_mce.md

```diff
@@ -24,41 +24,54 @@ $ oc get mce multiclusterengine-sample -ojsonpath="{.spec.overrides.components[?
 true
 ```
 
+If the value is `false`, make sure to set it true, and don't forget to change `mce-instance-name` to the name of your multicluster-engine instance:
+
+```bash
+oc patch mce <mce-instance-name> -n open-cluster-management --type=json -p='[{"op": "add", "path": "/spec/overrides/components/-","value":{"name":"hypershift-preview","enabled":true}}]'
+```
+
 ## Turn one of the managed clusters into the hypershift management cluster
 
-We call the cluster with the hypershift operator installed as the hypershift management cluster. In this section, we will use hypershift-addon to install a hypershift operator to one of the managed cluster.
+We call the cluster with the hypershift operator installed as the hypershift management cluster. In this section, we will use hypershift-addon to install a hypershift operator to one of the managed cluster. This step is only needed **once**, not needed for creating more clusters afterwards.
+
+Before creating the add-on, we need to make to provide the details of the s3 bucket where HyperShift will be storing OIDC discovery information. The s3 bucket is a pre-req for the installation
 
-1. Create ManagedClusterAddon hypershift-addon
+1. Create an oidc S3 credentials secret for the hypershift operator, name is `hypershift-operator-oidc-provider-s3-credentials` in the `hypershift-management-cluster` namespace, which one you want to install hypershift operator.
```

Comment:
Sorry, I know you just copied and pasted it, but since you're at it, could be change oidc => OIDC? Thanks.